docs/source/pages/documentation.md

```diff
@@ -1,18 +1,170 @@
+
+#### Creating *subject* and *session* folders
+
+In a typical neuroscience experiment, a data-collection session begins by creating the folder for the current subject name (e.g. mouse, rat) and current session name. Once created, the data for this session is stored in the created folder.
+
+The command `make-sub-folders` can be used automatically create folder trees that adhere to the SWC-Blueprint (and correspondingly, BIDS) specification. The linked specifications contain more detail, but at it's heard this requires:
+
+All subjects are given a numerical (integer) number that is prefixed with the key `sub-`. All sessions are also given a numerical (integer) number that is prefixed with the key `ses-`.
+
+Following this, optional information can be included in the form of key-value pairs. For example, a folder tree for *subject 1*, *session 1*  with *behavioural* data that includes the date of each session in the *session* folder name would be:
+
+```
+└── sub-001/
+    └── ses-001_date_20230516/
+        └── behav/
+            └── sub-001_ses-001_recording.mp4
+```
+
+In DataShuttle, this folder tree (excluding the .mp4 file which must be saved using third-party software) can be created (assuming today's date is `20220516`), with the command
+
+```
+datashuttle \
+my_first_project \
+make-sub-folders -sub 001 -ses 001_@DATE@ -dt behav
+```
+
+The leading `sub-` or `ses-` is optional when specifying folders to create. It is possible to automatically create date, time or datetime key-value pairs with the days `@DATE@`, `@TIME@` or `@DATETIME@` repsectively.
+
+Another example call, which creates a range of subject and session folders, is shown below:
+
+```
+datashuttle \
+my_first_project \
+make-sub-folders -sub 001@TO@003 -ses 010_@TIME@ -dt all
+```
+
+When the `all` argument is used for `--data_type` (`-dt`), the folders created depend on the *data-types* specified during *configuration* setup. For example, if
+`--use_behav`, `--use_funcimg`, `--use_histology` were set during *configuration* setup, the folder tree from the above command (assuming the time is 4.02.48 PM), would look like:
+
+```
+├── sub-001/
+│   ├── ses-010_time-160248/
+│   │   ├── behav
+│   │   └── funcimg
+│   └── histology
+├── sub-002/
+│   ├── ses-010_time-160248/
+│   │   ├── behav
+│   │   └── funcimg
+│   └── histology
+└── sub-003/
+    ├── ses-010_time-160248/
+    │   ├── behav
+    │   └── funcimg
+    └── histology
+```
+
+# FOR FULL DETAILS, SEE XXX
+
+
+#### Data Transfer
+
+
+
+#### Configuration
 
-On first setup, it is necessary to specify the project name, paths to the local project folder (typically empty on first use, on a
-local filesystem), and paths to central project folder and the connection method.
 
 This documentation gives examples both using the API (in the python console) or using the command line interface (in system terminal).
 
+#### C
```

Comment:
Is the addition of this line on purpose?

Reply:
No 😅